code/style.css

```diff
@@ -1,22 +1,80 @@
+/* Between line 2-14 and also main-container class, serve purpose of keeping the footer at the bottom of the page, didn't want a sticky one */
+* { 
+  box-sizing: border-box;
+ }
+ *:before,
+ *:after {
+  box-sizing: border-box;
+ }
+
+html,
+body {
+ height: 100%;
+ position: relative;
+}
+
 body {
   margin: 0;
   font-family: "Courier New", Courier, monospace;
+  background-image: url(./assets/background-img.jpg);
+  background-size: cover;
 }
 
-p {
-  margin: 1px;
-  font-weight: 300;
-  background-color: #fff;
+.main-container {
+  min-height: 100vh; /* will cover the 100% of viewport */
+  overflow: hidden;
+  display: block;
+  position: relative;
+  padding-bottom: 50px; /* height of the footer */
+}
+
+
+h1 {
+  font-size: 100px;
+  margin-top: 10px;
+  margin-bottom: 0;
+  font-family: 'Carter One', cursive;
```

Comment:
You have written "font-family: 'Carter One', cursive;" on line 36, 44, 64, 119. have you considered to write the font-family in the body so all of them can have the same font with just one line of code? And only have another code for "p".